# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../webcdwriter/hotfix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- webCDwriter-2.6.7/CDWserver/Store.cpp	2003-07-05 13:20:11.000000000 +0200
+++ webCDwriter-2.6.7-t2/CDWserver/Store.cpp	2005-01-14 10:12:36.848763864 +0100
@@ -32,7 +32,7 @@
 
 	maxOpenSessions = config.getMaxOpenSessions();
 	sessionsNum = 0;
-	sessions = new (Session *)[maxOpenSessions];
+	sessions = new Session *[maxOpenSessions];
 	int i;
 	for (i = 0; i < maxOpenSessions; i++)
 		sessions[i] = NULL;
